README.md

```diff
@@ -117,18 +161,79 @@ modes, bounded and unbounded.
 ```xml
 <dependency>
   <groupId>com.google.cloud.flink</groupId>
-  <artifactId>flink-connector-bigquery</artifactId>
-  <version>0.1.0-preview</version>
+  <artifactId>flink-1.17-connector-bigquery</artifactId>
+  <version>0.2.0-preview</version>
 </dependency>
 ```
 
 #### Relevant Files
 
+* Sink factory methods are defined at `com.google.cloud.flink.bigquery.sink.BigQuerySink`.
+* Sink configs are defined at `com.google.cloud.flink.bigquery.sink.BigQuerySinkConfig`.
 * Source factory methods are defined at `com.google.cloud.flink.bigquery.source.BigQuerySource`.
 * Source configs are defined at `com.google.cloud.flink.bigquery.source.config.BigQueryReadOptions`.
 * BigQuery connection config is defined at `com.google.cloud.flink.bigquery.common.config.BigQueryConnectOptions`.
 * Sample Flink application using connector is defined at `com.google.cloud.flink.bigquery.examples.BigQueryExample`.
 
+### At Least Once Sink
+
+Flink [Sink](https://nightlies.apache.org/flink/flink-docs-release-1.17/api/java/org/apache/flink/api/connector/sink2/Sink.html) 
+is the base interface for developing a sink. With checkpointing enabled, it can offer at-least-once consistency. Our 
+implementation uses BigQuery Storage's [default write stream](https://cloud.google.com/bigquery/docs/write-api#default_stream) 
+in Sink's [Writers](https://nightlies.apache.org/flink/flink-docs-release-1.17/api/java/org/apache/flink/api/connector/sink2/SinkWriter.html).
+
+```java
+// StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+// env.enableCheckpointing(checkpointInterval);
```

Comment:
are these supposed to be commented out?

Reply:
Yes. They've been mentioned just to point out what `env` variable means (since it is expected in Sink) and that checkpointing must be enabled (also mentioned in sink's usage description).

Reply:
Resolving this comment slightly prematurely. Please lmk if you'd like to see further changes. I can open up another PR.